api/v1alpha1/hivelocitymachine_types.go

```diff
@@ -99,6 +101,26 @@ type HivelocityMachineSpec struct {
 	Status ControllerGeneratedStatus `json:"status,omitempty"`
 }
 
+// DeviceSelector specifies matching criteria for labels on Devices.
+// This is used to limit the set of Device objects considered for
```

Comment:
This is used to target a specific set of devices that can be claimed by the HivelocityMachine